15_3sum.py

```diff
@@ -0,0 +1,106 @@
+from typing import List
+
+class Solution:
+    def threeSum(self, nums: List[int]) -> List[List[int]]:
+        """
+                https://leetcode.com/problems/3sum/
+                // Time Complexity : O(nlogn + n^2)
+                // Space Complexity : O(n)
+                            Sorting requires O(n) space
```

Comment:
You are right although we are not sure what is the sorting algo used under the hood.  Keep it generalized